pkg/rsyslog/rsyslog.conf

```diff
@@ -55,12 +55,14 @@ action(type="mmjsonparse" cookie="")
 
```

Comment:
This is not really needed for this PR, right? I mean it could be a fine change in its own PR but not required for the NIM part, correct @gkodali-zededa ?

Reply:
@rvs Yes, this is not required for this PR but the I saw this also at the same time. Should I remove this change and raise a new PR?

Reply:
@rvs I backed out the rsyslog related commit. Will send as part of another PR.